share_extension/android/extension_channels/extension_channels.js

```diff
@@ -238,7 +309,7 @@ export default class ExtensionTeam extends PureComponent {
 const getStyleSheet = makeStyleSheetFromTheme((theme) => {
     return {
         flex: {
-            flex: 1,
+            flexGrow: 0,
```

Comment:
Won't `flex: 1` fill the allocated space, effectively doing the same "no resize" action as `flexGrow: 0` above?

My guidance is based on the footnote in the official spec of `flexGrow` (see [here](https://drafts.csswg.org/css-flexbox/#flex-grow-property)) which favours `flex: n` over `flexGrow`. But perhaps there is a legitimate use case/side-effect being handled in your case here which I'd like to understand better?